### ISSUE TITLE
podrasowanie auto-maila z rejestracji

### ISSUE THREAD

**Issue by ducin** [PyWaw/summit.pywaw.org#12]
Chciałbym zmienić treść auto-maila, a w zasadzie jego stopkę:
- dodać nasze summit-media (twitter, facebook, link do storny eventu, etc.)
- dodać logo (grafikę) do stopki
  KH wspominał, że mamy ładny przygotowany template w mail**impie i można go re-użyć.

**Comment by konradhalas** [comment]
Tak, proponuję zamienić tego maila na HTML'owego, będzie jak znalazł do innych maili.
